plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -816,8 +886,23 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 			g_prefix_error(error, "failed to get vendor support info from device: ");
 			return FALSE;
 		}
+		if (!fu_genesys_usbhub_device_get_descriptor_data(
+			vendor_buf,
+			(guint8 *)&self->vs_ts,
+			sizeof(FuGenesysVendorSupportToolString),
+			error)) {
+			g_prefix_error(error, "failed to get vendor support info from device: ");
+			return FALSE;
+		}
+		if (g_getenv("FWUPD_GENESYS_USBHUB_VERBOSE") != NULL) {
+			fu_common_dump_raw(G_LOG_DOMAIN,
+					   "Vendor support",
+					   (guint8 *)&self->vs_ts,
+					   sizeof(FuGenesysVendorSupportToolString));
+		}
 	}
 
+	/* setup cfi device */
```

Comment:
Make sense to space the code.

That comment would go before `self->cfi_device = fu_genesys_usbhub_device_cfi_setup(self, error);` with an empty line right before to space the blocks:

The following would make me much more happy :wink: 
```
if (!fu_genesys_usbhub_device_set_isp_mode(self, ISP_ENTER, error))     
        return FALSE;

/* setup cfi device */
self->cfi_device = fu_genesys_usbhub_device_cfi_setup(self, error);
```